### ISSUE TITLE
Profiles table rows should have role "checkbox" and check on space

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#214623]
The profiles table rows should have the accessibility role "checkbox"  and pressing "space" should toggle the checkbox. Similar how breakpoints list works.

![Image](https://github.com/microsoft/vscode/assets/1926584/871c4de6-71c0-4844-9dec-5e7bffee6286)

**Comment by sandy081** [comment]
> The profiles table rows should have the accessibility role "checkbox" 

There are more actions than checkbox on the row and toggling Use Default Profile checkbox is not a primary action. Is this expected to have accessibility role and toggle on space still?

**Comment by isidorn** [comment]
Aha so you are saying that "open to the side" is the primary action. In that case we do not have to update the role.
If you think having "open to the side" as primary is the best experience then feel free to close this one.
